src/main/java/seedu/address/model/person/Name.java

```diff
@@ -9,15 +9,8 @@
  * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
  */
 public class Name {
-
-    public static final String MESSAGE_NAME_CONSTRAINTS =
-            "Person names should only contain alphanumeric characters and spaces, and it should not be blank";
-
-    /*
-     * The first character of the address must not be a whitespace,
-     * otherwise " " (a blank string) becomes a valid input.
-     */
-    public static final String NAME_VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    //@@author alexfoodw
+    public static final String MESSAGE_NAME_CONSTRAINTS = "Person names should not be blank";
```

Comment:
Why do we omit the rest of the regex?

Reply:
ok cause facebook has some chinese names which is not in ASCII hahaha so i just decided to remove it entirely